homeassistant/components/ecobee/sensor.py

```diff
@@ -141,5 +165,6 @@ async def async_update(self):
             for item in sensor["capability"]:
                 if item["type"] != self.entity_description.key:
                     continue
-                self._state = item["value"]
+                thermostat = self.data.ecobee.get_thermostat(self.index)
```

Comment:
Is there a reason that you're not setting to / using `self.thermostat` here?

Reply:
Thought I got them all...  Guess not.